internal/tui/item.go

```diff
@@ -219,259 +218,3 @@ func (i *Item) tasksView() string {
 
 	return strings.Join(tasks, "\n")
 }
-
```

Comment:
Sorry for the noise, I don't think anything actually changed here, just wanted to split it out.